src/controllers/datalake.ts

```diff
@@ -1,16 +1,9 @@
 import { basename } from 'path';
 import { Readable } from 'stream';
 
-import * as dotenv from 'dotenv';
 import { DataLakeServiceClient, StorageSharedKeyCredential } from '@azure/storage-file-datalake';
-import pino from 'pino';
 
-dotenv.config();
-
-export const logger = pino({
```

Comment:
We'll need to add a child logger here.  I'll raise an issue for this so that we can go back to it rather than blocking this PR from moving forward

Reply:
done 👍